website/layouts/partials/favicons.html

```diff
@@ -0,0 +1,4 @@
+<head>
```

Comment:
Did it not cooperate out of the box once the favicons were in the right directory? The [Docsy docs](https://www.docsy.dev/docs/adding-content/iconsimages/#add-your-favicons suggest that you only need the partial if you want to customize

Reply:
I tried dropping the favicon.ico and favicon.svg in static/favicons/ and it didn't work for me. Looking at the theme source (docsy/layouts/partials/favicons.html) it appears they only support .ico and .png files which means we couldn't do dark mode/light mode switching or svgs.

Reply:
Good to know, I'm fine including the partial, just wanted to understand the rationale.